pkg/db/db.go

```diff
@@ -2691,17 +2724,15 @@ func (h *DBHandler) DBWriteEnvironmentLockInternal(ctx context.Context, tx *sql.
 	insertQuery := h.AdaptQuery(
 		"INSERT INTO environment_locks (eslVersion, created, lockID, envName, deleted, metadata) VALUES (?, ?, ?, ?, ?, ?);")
 
-	var timetoInsert time.Time
-	if useTimeInLock {
-		timetoInsert = envLock.Created
-	} else {
-		timetoInsert = time.Now().UTC()
+	now, err := h.DBReadTransactionTimestamp(ctx, tx)
```

Comment:
what about the `usetimeinlock` parameter?

Reply:
The usetimeinlock parameter now does not make sense. If we want to make the timestamps consistent, we can't have a different reference for locks. The created/timestamp column should always have the timestamp of insertion into the database. If this isn't the case, then locks will not work when we try to backtrack to a previous DB state. However, I would argue we should still store it on the metadata